src/open_samus_returns_rando/specific_patches/cosmetic_patches.py

```diff
@@ -0,0 +1,36 @@
+from mercury_engine_data_structures.formats.bmtun import Bmtun
+from open_samus_returns_rando.misc_patches import lua_util
+from open_samus_returns_rando.patcher_editor import PatcherEditor
+
+
+def patch_cosmetics(editor: PatcherEditor, configuration: dict):
+    tunables = editor.get_file("system/tunables/tunables.bmtun", Bmtun)
+    _tunable_cosmetics(tunables, configuration)
+    _lua_cosmetics(configuration)
```

Comment:
This line is doing nothing. It gives you the whole lua file as a string but you never assign it to a variable and just throw it away.
It's not needed because you do that in the part where my other comment is.

Reply:
Is it not calling to the replacement below?

Reply:
It's calling it but the only thing the replacement really does is returning a string with what you need to do something. As the return is not even assigned, I can easily tell this is doing (logically) nothing.